common/customtypes/pages/index.json

```diff
@@ -42,6 +42,15 @@
           "placeholder": ""
         }
       },
+      "featuredText": {
+        "type": "StructuredText",
+        "config": {
+          "label": "Featured text",
+          "placeholder": "Introductory/important text to appear near the top of the page",
+          "allowTargetBlank": false,
+          "multi": "paragraph,strong,em,hyperlink"
+        }
+      },
```

Comment:
It could be nice to discuss what we want this kind of field to be called and try to use the same across ([as discussed in the RFC for the "All" search](https://github.com/wellcomecollection/docs/tree/2e78c52e2df5be9c6691e3b3ce33b7b04800735d/rfcs/062-content-api-all-search#description-captions-standfirsts-and-intro-texts), I think we use a few different names or methods for this). 
This feels like an opportunity? (Not to change it for all in this PR! Just to make a call and use that decision to name this field?)

eg. exhibitions guides use "intro_text".

Reply:
Good point. There is also the `standfirst` which is another thing that can be a slice in the `body` but where it doesn't feel like it makes sense to be there. I suppose that from an editorial standpoint they could be _most_ appropriately named something different in each case, but as far as the code is concerned, maybe we just need to find the least abstract term than encompasses, 'intro text', 'standfirst' and 'featured text' (my gut feeling is that 'intro text' covers all three better than either of the other two)

Reply:
I copied the `camelCasing` that I saw in Slice Machine for exhibition guides? But there does seem to be some `intro_text`s kicking about too. I definitely favour camelCase (I think our linter even complains if we don't use it?)

<img width="653" alt="image" src="https://github.com/user-attachments/assets/54b7efea-ca32-490a-b5e3-56b0fabcbc20">

Reply:
Might get modified in the transformer then 🤔 thanks for looking!